Converters/ArcGIS/Speckle.Converters.ArcGIS3/Utils/LocalToGlobalConverterUtils.cs

```diff
@@ -30,100 +31,82 @@ public Base TransformObjects(Base atomicObject, List<Matrix4x4> matrix)
 
     // This is a temp hack. We would add transformations to conversions later instead try to copy objects like this.
```

Comment:
If we consider this a hack we should add a linear ticket to unhack it otherwise it will stay a hack